packages/app/src/components/choropleth/safety-region-choropleth.tsx

```diff
@@ -113,26 +115,50 @@ export function SafetyRegionChoropleth<T, K extends RegionsMetricName>(
     [getFillColor, hasData]
   );
 
+  const highightCallback = useCallback(
+    (feature: Feature<MultiPolygon, SafetyRegionProperties>, path: string) => {
+      const { vrcode } = feature.properties;
+
+      if (highlightCode !== vrcode) return;
+
+      return (
+        <>
+          <Path
+            key={`${vrcode}-outside-border`}
+            d={path}
+            stroke="#fff"
+            strokeWidth={8}
+          />
+          <Path
+            key={`${vrcode}-inside-border`}
+            d={path}
+            stroke="#000"
+            strokeWidth={2}
+          />
+        </>
```

Comment:
I think the `key`-props can be moved to the fragment
```suggestion
        <Fragment key={vrcode}>
          <Path
            d={path}
            stroke="#fff"
            strokeWidth={8}
          />
          <Path
            d={path}
            stroke="#000"
            strokeWidth={2}
          />
        </Fragment>
```